### PR TITLE
add package-lock.json to .gitignore

### PR DESCRIPTION
**Pull request by hwanseung**
package-lock.json is automatically generated for any operations
where npm modifies either the node_modules tree, or package.json.
so sometimes it might be made to bother when use git command.

ISSUE=None

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .last_update
+package-lock.json
```
Comment (romandev) [inline .gitignore:2]: Can we keep an alphabetical order?

**Review by romandev — COMMENTED** [review]
Thank you for this change.

File: .gitignore (hunk 1 above)
Comment (hwanseung) [reply .gitignore]: i thought ordering is first file and then folder. so i added it before folder's line.
do we keep an alphabetical order without distinguish file or folder?

Comment (hwanseung) [reply .gitignore:2]: i thought ordering is first file and then folder. so i added it before folder's line.
do we keep an alphabetical order without distinguish file or folder?

Comment (romandev) [reply .gitignore:2]: acknowledgement.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**